scalafix/rules/src/main/scala/fix/v0_14_0/FixAvroCoder.scala

```diff
@@ -17,6 +17,7 @@ object FixAvroCoder {
   )
   val AvroMatcher: SymbolMatcher = SymbolMatcher.normalized(
     "org/apache/avro/specific/SpecificRecord",
+    "org/apache/avro/specific/SpecificRecordBase",
```

Comment:
For context, I've seen several user repos that use type bound `T <: SpecificRecordBase`

(maybe we should also refactor that type bound from `T <: SpecificRecordBase` => `T <: SpecificRecord`?)

Reply:
> maybe we should also refactor
if it's not a ton of scalafix code :) generally a more relaxed type constraint is better, but is there any other argument?